images/build/yarn-cache/cacheNodePackages.sh

```diff
@@ -203,8 +203,11 @@ for pkg in \
     installPackage $pkg
 done;
 
-# Open up the cache so other users can consume it.
-chmod -R 777 $YARN_CACHE_DIR
-
 echo "Caching done. Total cache size: $(du -sh $YARN_CACHE_DIR | cut -f -1)"
-rm -fr $TEMP_APP_DIR
\ No newline at end of file
+rm -fr $TEMP_APP_DIR
+
+mkdir -p /output
+cd $YARN_CACHE_DIR
+tar -zcf /output/yarncache.tar.gz .
```

Comment:
here we are compressing whole directory which is $YARN_CACHE_DIR ... is it also happen to be /output?

Reply:
basically i am establishing a pattern where either this yarn-cache or the 'intermediate' stage has an folder called 'output' where we can find the stuff to be copy from.